testing/benchmark/pubspec.yaml

```diff
@@ -2,10 +2,8 @@ name: flutter_engine_benchmark
 
 dependencies:
   git: any
-  metrics_center:
-    # TODO(liyuqian): once metrics_center is properly reviewed, add it to
-    # flutter/packages, publish on pub.dev, and use the published package here.
-    git: https://github.com/liyuqian/metrics_center.git
+
+  metrics_center: ^0.0.4
```

Comment:
Is there any risk of builds not being reproducible if this isn't pinned to a specific version?

Reply:
builds or benchmark runs, that is.

Reply:
Probably. Now the version is pinned. Thanks for the suggestion!